docs/user/README.md

```diff
@@ -500,11 +500,13 @@ In any cursorless command, the action defines what happens to the specified targ
 
 Despite the name cursorless, some of the most basic commands in cursorless are for moving the cursor.
 
-Note that when combined with list targets, these commands will result in multiple cursors
+Note that when combined with list targets, `take`/`pre`/`post` commands will result in multiple cursors.
 
-- `"take <TARGET>"`: Selects the given target
-- `"pre <TARGET>"`: Places the cursor before the given target
-- `"post <TARGET>"`: Places the cursor after the given target
+- `"take <TARGET>"`: Selects the given target.
+- `"give <TARGET>"`: Deselects the given target.
+  Will not go below one selection.
```

Comment:
not sure we need a line for this one; they'll get an error if they ever try, and that's not really a Cursorless limitation so much as a VSCode limitation

```suggestion
```